rebasehelper/utils.py

```diff
@@ -141,6 +145,103 @@ def cprint(cls, message, fg=None, bg=None, style=None):
         else:
             print(message)
 
+    @staticmethod
+    def parse_rgb_device_specification(specification):
+        """Parses RGB device specification.
+        Args:
+            specification(str): RGB device specification.
+
+        Returns:
+            tuple: If the specification follows correct format, the first element is RGB tuple and the second is
+            bit width of the RGB. Otherwise, both elements are None.
+
+        """
+        match = re.match(r'^rgb:([A-Fa-f0-9]{1,4})/([A-Fa-f0-9]{1,4})/([A-Fa-f0-9]{1,4})$', str(specification))
+        if match:
+            rgb = match.groups()
+            bit_width = max([len(x) for x in rgb]) * 4
+            return rgb, bit_width
```

Comment:
It would make more sense to return the values as numbers. Otherwise you don't need to return bit-width, it's deducible from the strings.